src/integration/blockchain/monero/monero-client.ts

```diff
@@ -57,6 +63,26 @@ export class MoneroClient {
       .then((r) => this.convertFeeEstimateAuToXmr(r.result));
   }
 
+  // --- UNIMPLEMENTED METHODS --- //
+
+  async getToken(_: Asset): Promise<Currency> {
+    throw new Error('Monero has no token');
+  }
+
+  async getTokenBalance(_: Asset, __?: string): Promise<number> {
+    throw new Error('Monero has no token');
+  }
+
+  async getTokenBalances(_: Asset[], __?: string): Promise<BlockchainTokenBalance[]> {
+    throw new Error('Monero has no token');
+  }
+
+  async isTxComplete(_: string, __?: number): Promise<boolean> {
+    throw new Error('Monero method not implemented.');
+  }

```

Comment:
Implement it quickly?
```ts
const transaction = await this.getTransaction(txId);
return MoneroHelper.isTransactionComplete(transaction);
```
and use it everywhere, where `MoneroHelper.isTransactionComplete` is currently used.